### ISSUE TITLE
Scrollable one-page for /docs

### ISSUE THREAD

**Issue by joenash** [enso-org/enso-org.github.io#8]
### General Summary
Render all of a given submodule as a scrollable one-page. 

### Motivation
An alternate mode of viewing the page that facilitates long reads and vigorous ctrl-f'ing.

**Comment by iamrecursion** [comment]
The one page should be on a per-repo basis, I think. I don't think we want to merge developer docs from different repos into a single page, do you?

**Comment by joenash** [comment]
Agreed, sorry that's what I was going for with `of a given submodule`

**Comment by joenash** [comment]
Thinking on implementation, we could either generate a one pager for each directory in route, or add a frontmatter field to a README which we look for to then create a one pager from all the files/directories below it
